backend/apps/volontulo/serializers.py

```diff
@@ -204,7 +209,10 @@ def get_is_administrator(obj):
     def get_organizations(self, obj):
         """Returns organizations that user belongs to."""
         qs = obj.userprofile.organizations.all()
-        return OrganizationSerializer(qs, many=True, context=self.context).data
+        return OrganizationSerializer(qs, many=True, context=self.context).data\
+
+    def get_phone_no(self, obj):
```

Comment:
You missed a docstring here (and it cause Travis to fail: `C:214, 4: Missing method docstring (missing-docstring)`).

General docstring convention for Python is described in document named PEP-257: https://www.python.org/dev/peps/pep-0257/

Reply:
The other issue with that method is that it doesn't use `self` parameter (which point to class instance, which this method is executed at), we only use the second parameted `obj` - generally we can find two decorator usefull here:
* `classmethod` - see: https://docs.python.org/3/library/functions.html#classmethod
* `staticmethod` - see: https://docs.python.org/3/library/functions.html#staticmethod

Some introduction for beginners is availeble here: https://stackoverflow.com/a/12179752 and example usage of it You can find even in this file, see: https://github.com/CodeForPoznan/volontulo/blob/master/backend/apps/volontulo/serializers.py#L199